jumpscale/packages/tfgrid_solutions/chats/peertube.py

```diff
@@ -155,27 +155,7 @@ def reservation(self):
             raise StopChatFlow(f"Failed to create subdomain {self.domain} on gateway" f" {self.gateway.node_id} {_id}")
         self.threebot_url = f"https://{self.domain}"
 
-        # expose threebot container
-        _id = deployer.expose_address(
-            pool_id=self.pool_id,
-            gateway_id=self.gateway.node_id,
-            network_name=self.network_view.name,
-            local_ip=self.ip_address,
-            port=80,
-            tls_port=443,
-            trc_secret=self.secret,
-            node_id=self.selected_node.node_id,
-            reserve_proxy=True,
-            domain_name=self.domain,
-            solution_uuid=self.solution_id,
-            **self.solution_metadata,
-        )
-        success = deployer.wait_workload(_id, self)
-        if not success:
-            # solutions.cancel_solution(self.workload_ids)
-            raise StopChatFlow(f"Failed to create trc container on node {self.selected_node.node_id}" f" {_id}")
-
-        entrypoint = f'/usr/local/bin/startup.sh "{self.domain}" "{self.email}"'
+        entrypoint = f'/usr/local/bin/startup.sh "{self.domain}"'
```

Comment:
Why did you remove the email parameter from the command?

Reply:
The only use for the email was to generate the certificate now nginx container creates it.